### DISCUSSION TITLE
How to autocomplete user handles for mentioning

### DISCUSSION THREAD

**Discussion by braineo** [magit/forge#413] [Support] [answered]
Hi, all. I started to do discussion inside forge and it is awesome!

For issues and PR, I noticed that I can choose a assignee from a list. While in the comment/discussion, it seems no such autocomplete for user handle.

What method can I manually invoke to get the names, or I need to write something like this function?

```lisp
(defun forge-edit-topic-assignees (n)
  "Edit the assignees of the current topic.
If there is no current topic or with a prefix argument read a
topic N and modify that instead."
  (interactive (list (forge-read-topic "Edit assignees of")))
  (let* ((topic (forge-get-topic n))
         (repo  (forge-get-repository topic))
         (value (closql--iref topic 'assignees))
```

**Comment by tarsius** [comment, marked as answer]
This would have to be implemented using a function similar to `forge-topic-completion-at-point` and adding that to `completion-at-point-functions` for the appropriate mode.

**Reply by braineo** [reply]
Thank you! Let me try it out.
